.eslintcache

```diff
@@ -0,0 +1 @@
ages":"31","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"16iblvo",{"filePath":"32","messages":"33","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"34","messages":"35","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"36","messages":"37","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"38","messages":"39","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"40","messages":"41","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"42","messages":"43","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"44","messages":"45","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"46","messages":"47","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"48","messages":"49","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"50","messages":"51","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"52","messages":"53","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"54","messages":"55","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"56","messages":"57","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\index.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\rootReducer.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\views\\Home.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\views\\ErrorPages\\Page500.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\features\\draftStatusSlice.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\features\\wizardObjectTypeSlice.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\features\\wizardSubmissionFolderSlice.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardSteps\\WizardAddObjectStep.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardComponents\\WizardHeader.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardForms\\WizardStatusMessageHandler.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardComponents\\WizardSavedObjectsList.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\services\\folderAPI.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardComponents\\WizardAlert.js",[],"D:\\Code\\csc\\allas\\metadata-submitter-frontend\\src\\components\\NewDraftWizard\\WizardForms\\WizardAjvResolver.js",[]]
```

Comment:
this looks like something for gitignore or am i wrong ?

Reply:
Added to gitignore and deleted file.